NEWS.md

```diff
@@ -1,6 +1,7 @@
 rmarkdown 2.29
 ================================================================================
 
+- A parameter allowing multiple selections uses a select input control by default regardless of the number of choices.
```

Comment:
Users probably have no idea what this short sentence means.

````suggestion
- Added an option `multiple` for parameters with `choices` to use the `select` or `radio` input control regardless of the number of choices. Previously, the `select` input is used when the number of choices is greater than 4, otherwise the `radio` input is used. Now `select` will be used when `multiple` is true, and `radio` is used when `multiple` is false, e.g.,

  ---
  params:
    foo:
      choices: ["a", "b", "c"]
      multiple: true  # use select input 
    bar:
      choices: ["a", "b", "c", "d", "e"]
      multiple: false  # use radio input 
  ---
  ```
````

Reply:
We have long supported "multiple". Previously, we made the wrong choice when there were a small number of choices and multiple were allowed.

This is the one test case that demonstrates the change:

```r
  expect_equal(params_get_input(list(
    value = "red",
    multiple = TRUE,
    choices = c("red", "green", "blue")
  )), "select")
```

Previously, the small number of choices causes us to inferred a "radio" input without recognizing the `multiple = TRUE` constraint.

I've pushed a NEWS update, but it feels too wordy for such a small, subtle change. I defer to your judgement.

Reply:
Oh I see. Thanks for the explanation! I have totally forgotten the `multiple` option.